converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/DicomWeb/Service/ChangeFeedRetrieveService.cs

```diff
@@ -39,11 +39,6 @@ public async Task<IReadOnlyList<ChangeFeedEntry>> RetrieveChangeFeedAsync(long o
 
         ChangeFeedEntry[] changeFeedEntries = await result.ToArrayAsync(cancellationToken);
```

Comment:
looks like you don't need this line anymore since we aren't using changefeedentries

Reply:
Removed unnecessary line